src/Providers/Discord.php

```diff
@@ -11,7 +13,13 @@ public function __construct(
     ) {
     }
 
-    public function send($url, $params, $webhook)
+    /**
+     * @param $url
+     * @param $params
+     * @param $webhook
+     * @throws \Exception
+     */
+    public function send($url, $params, $webhook): \GuzzleHttp\Promise\PromiseInterface|\Illuminate\Http\Client\Response
```

Comment:
response type doesn't match with the definition in interface. 
thus I'd prefer to not force return type in `send` method